packages/special-pages/pages/duckplayer/src/js/index.js

```diff
@@ -123,8 +156,15 @@ const VideoPlayer = {
     onIframeLoaded: (callback) => {
         const iframe = VideoPlayer.iframe()
 
-        if (iframe) {
-            iframe.addEventListener('load', callback)
+        if (VideoPlayer.loaded) {
+            callback()
+        } else {
+            if (iframe && !VideoPlayer.loaded) {
```

Comment:
Surplus check based on the if conditional.
```suggestion
            if (iframe) {
```

Reply:
nice, thanks!